### PR TITLE
support using separate fargate profile for each namespace

### PR DESCRIPTION
**Pull request by schollii**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -34,7 +34,7 @@ data "aws_iam_policy_document" "assume_role" {
 
 resource "aws_iam_role" "default" {
   count              = var.enabled ? 1 : 0
-  name               = module.label.id
+  name               = "${module.label.id}-${var.kubernetes_namespace}"
```
Comment (osterman) [inline main.tf:37]: @Nuru is this where we should use `${module.label.id}@${var.kubernetes_namespace}` following our current convention?

**Comment by schollii** [comment]
@aknysh and @Nuru I looked at the test failures and I'm not seeing anything that relates to the change I made, maybe it needs to be rebased? any other ideas?

**Comment by osterman** [comment]
Aha, test failures here are due to out-of-date tests. We need to upgrade to go modules. I'm happy to give some directives here how to do that, or you can wait for us (which might be a while). 

Are you on [slack](https://slack.cloudposse.com)? if so, ping me there. I'm faster to respond.

**Pull request closed without merging**